p4include/v1model.p4

```diff
@@ -133,14 +145,17 @@ parser Parser<H, M>(packet_in b,
                     inout standard_metadata_t standard_metadata);
 control VerifyChecksum<H, M>(in H hdr,
                              inout M meta);
+@pipeline
```

Comment:
The @pipeline annotation is not documented anywhere. I think it should be removed.
You are essentially duplicating functionality provided by the package operation - the fact that something is a pipeline is given by the fact that it is used as an argument to the V1Switch package in the right position.